### DISCUSSION TITLE
FLYCAST WIN7

### DISCUSSION THREAD

**Discussion by Juissi71** [flyinghead/flycast#1468] [General]
Hi, is there any tips to get Flycast working on Windows 7 64bit?

**Comment by flyinghead** [comment]
what's the issue?

**Reply by Juissi71** [reply]
I'll check it tonight, but it't won't start in my Arcade cabinet and my desktop (both Win 7 64). Mame, Sega model 2&3, Demul emulators works.

**Reply by flyinghead** [reply]
try to get some logs by enabling "Settings > Advanced > Log to File"

**Reply by Juissi71** [reply]
Ok, thank you

**Reply by Juissi71** [reply]
Emulator won't start at all. The application was unable to start correctly (0x000007b).
I checked with win10 desktop that DX9 is checked

**Reply by cinnamonmatexfce** [reply]
> Emulator won't start at all. The application was unable to start correctly (0x000007b).
> I checked with win10 desktop that DX9 is checked

Maybe related to this?: https://github.com/flyinghead/flycast/issues/359#issuecomment-921678195

**Reply by flyinghead** [reply]
You probably have some 32-bit DLL that's being loaded instead the 64-bit one. Check the issue referenced by @cinnamonmatexfce above and try to reinstall GPU drivers, DirectX runtimes, etc.

**Reply by Juissi71** [reply]
DX things didn't help, I'll continue after easter

**Reply by Juissi71** [reply]
![image](https://github.com/flyinghead/flycast/assets/164998729/637cdcd0-f256-4d52-b90c-0d6a73df213f)
Do I'm missing some drivers?
![Capture](https://github.com/flyinghead/flycast/assets/164998729/e154b1f7-1fef-478f-b7af-48debf0afa11)

**Reply by Juissi71** [reply]
Dependency walker was THE application. With DW I checked missing / wrong DLL. Now Flycast is working
